app/models/content_store_organisation.rb

```diff
@@ -0,0 +1,99 @@
+require 'active_model'
+
+class ContentStoreOrganisation
+  include ActiveModel::Model
+
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def live?
+    organisation_state == 'live'
+  end
+
+  def self.find!(base_path)
+    content_item = ContentItem.find!(base_path)
+    new(content_item)
+  end
+
+  def number_10
+    details["ordered_executive_offices"]
+  end
+
+  def number_10?(organisation_type)
+    organisation_type == "Executive offices"
+  end
+
+  def ministerial_departments
+    details["ordered_ministerial_departments"]
+  end
+
+  def non_ministerial_departments
+    details["ordered_non_ministerial_departments"]
+  end
+
+  def ordered_agencies_and_other_public_bodies
+    details["ordered_agencies_and_other_public_bodies"]
+  end
+
+  def ordered_high_profile_groups
+    details["ordered_high_profile_groups"]
+  end
+
+  def ordered_public_corporations
+    details["ordered_public_corporations"]
+  end
+
+  def ordered_devolved_administrations
+    details["ordered_devolved_administrations"]
+  end
+
+  def count_by_type(organisation_type)
+    details[organisation_type].count
+  end
+
+  def organisations_list(department_type)
+    details[department_type]
+  end
+
+  def organisation_department_count(organisation)
+    organisation["works_with"].values.flatten.count
+  end
+
+  def get_name_of_organisation(organisation_type)
+    org_name = {
+        "ordered_executive_offices" => "",
+        "ordered_ministerial_departments" => "Ministerial departments",
+        "ordered_non_ministerial_departments" => "Non ministerial departments",
+        "ordered_agencies_and_other_public_bodies" => "Agencies and other public bodies",
+        "ordered_high_profile_groups" => "High profile groups",
+        "ordered_public_corporations" => "Public corporations",
+        "ordered_devolved_administrations" => "Devolved administrations"
+    }
+    org_name[organisation_type]
+  end
+
+  def get_name_of_department_category(category_type)
```

Comment:
Using `get_` is not particularly rubyish.  It reads better (in the view for example) if we use something like
```
@organisations.category_name(category_type)
```
We should also move the hash outside of this method and make it a constant - otherwise we instantiate it every time we call the method.

Reply:
👍